spec/orders.mediawiki

```diff
@@ -582,29 +584,65 @@ the checksum generated from their local copy of the epoch queue.
 ! field   !! size (bytes)  !! description
 |-
 | pimg    || string || hex-encoded preimage for the order's commitment
+|-
+| ackid   || int || the message ID to be used for the server's preimage acknowledgement
 |}
 
-==Match negotiation==
+The server will send an [[comm.mediawiki/#Acknowledgements|acknowledgement]] of
+their receipt of the preimage. The message ID of the acknowledgement will be set
+to the specified <code>ackid</code>. The signature message is the byte-encoded
+preimage.
```

Comment:
It seems to me that if the server wanted to exclude an order from the shuffle, they would just not ack, so it's not clear if this adds a lot unless we adapt it to communicate error conditions with the response handling.

Reply:
> unless we adapt it to communicate error conditions with the response handling

That is the purpose here.